### PR TITLE
Scale gl3d error bars

### PR DESCRIPTION
**Pull request by etpinard**
@bpostlethwaite @alexcjohnson 

When added support for 3d date time axes using a [linear map](https://github.com/plotly/plotly.js/blob/master/src/traces/scatter3d/convert.js#L186), we *forgot* to make 3d error bars obey the same transformation. 3d error bars have been broken even since. Luckily this will never happen again thanks too 972461f00bcc338db65faa32f25d7fe2e012c02b .

### CODE REVIEW

File: test/image/mocks/gl3d_errorbars_zx.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                    500,
+                    1000
+                ],
+                "arrayminus": [
+                    0,
+                    500,
+                    0
+                ],
+                "color": "green",
+                "thinkness": 5
```
Comment (alexcjohnson) [inline test/image/mocks/gl3d_errorbars_zx.json]: I don't "think" that's what you meant

File: test/image/mocks/gl3d_errorbars_zy.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            "z": [
+                200,
+                100,
+                200
+            ],
+            "error_z": {
+                "type": "percent",
+                "value": 20,
+                "color": "red",
+                "width": 0.5
```
Comment (alexcjohnson) [inline test/image/mocks/gl3d_errorbars_zy.json]: This is sposed to add caps, right? Want to make it bigger so we can see them and verify that they end up looking similar to caps that size on 2D svg error bars?

**Comment by alexcjohnson** [comment]
Nice catch. Maybe one more test image that covers sqrt and symmetric data?

File: test/image/mocks/gl3d_errorbars_zx.json (hunk 1 above)
Comment (etpinard) [inline test/image/mocks/gl3d_errorbars_zx.json]: haha good catch. Thanks.

File: test/image/mocks/gl3d_errorbars_zy.json (hunk 2 above)
Comment (etpinard) [inline test/image/mocks/gl3d_errorbars_zy.json]: sure. my mistake

**Comment by alexcjohnson** [comment]
:dancer: if you're going to make a separate PR to fix and test `sqrt`

**Pull request merged**